### ISSUE TITLE
Tile 11691(exps 267072 and 267073) has two missing petals for unknown reasons

### ISSUE THREAD

**Issue by ashleyjross** [desihub/desisurveyops#253]
Something(s) strange happened with tile 11691 (exps 267072 and 267073 the nightlog says everything was normal):
The nominal results have us losing two petals. One I guess because of only 2 standard stars (but I thought we had successful petals with only 2 in the past). The other is unclear; it looks like a total failure (e.g., like the whole petal dropped out), but the nightlog doesn't mention anything.
<img width="1183" alt="Screenshot 2024-12-07 at 3 44 30 PM" src="https://github.com/user-attachments/assets/92252124-2ee6-4ad0-93c9-6c5b36932a44">

**Comment by araichoor** [comment]
quickly:
- petal=5 has zero standard stars: https://data.desi.lbl.gov/desi/target/fiberassign/tiles/trunk/011/fiberassign-011691.png (I remark it is not in the list here: https://github.com/desihub/desisurveyops/issues/82#issuecomment-1347655891, but it could be fiber status changed)
- petal=7: if I remember correctly, when there only are 2 std stars, the desispec code accepts those if they "agree", with some metrics.

**Comment by ashleyjross** [comment]
Right, probably the >0.1 RMS rejects the 2 standard petal 7 and the 0 standards on petal 5 explains everything there...I guess the question is whether to try to understand it more or accept it as is and move on...(maybe with trying to prevent future occurrences as an action item...)

**Comment by schlafly** [comment]
The low number of standards is characteristic of this part of the sky; I don't think there's more to learn here.

**Comment by geordie666** [comment]
OK, I'll reset the QA for tile `11691`. @ashleyjross: Tile `11691` should pop back up in tile QA today. Could you please mark it as "good"?
